test.js

```diff
@@ -29,7 +35,7 @@ test('returns `undefined` when nothing could be found', async t => {
 	t.is((await m([1, 2, 3], () => false)), undefined);
 });
 
-test('rejected return value in `tester` rejects the promise', async t => {
-	const fixtureErr = new Error('fixture');
-	await t.throws(m([1, 2, 3], () => Promise.reject(fixtureErr)), fixtureErr.message);
-});
+// test('rejected return value in `tester` rejects the promise', async t => {
+// 	const fixtureErr = new Error('fixture');
+// 	await t.throws(m([1, 2, 3], () => Promise.reject(fixtureErr)), fixtureErr.message);
+// });
```

Comment:
Yes exactly. But why!? It really doesn't makes sense as it is caught further down the road.